openwisp_radius/receivers.py

```diff
@@ -100,22 +100,27 @@ def close_previous_radius_accounting_sessions(instance, created, **kwargs):
     if not created or not instance.called_station_id:
         return
     RadiusAccounting = load_model('RadiusAccounting')
-    closed_session = []
+    closed_sessions = []
+    logger.info(f'username: {instance.username}; calling-station-id: {instance.calling_station_id}')
     open_sessions = RadiusAccounting.objects.exclude(
         unique_id=instance.unique_id
     ).filter(
         stop_time__isnull=True,
-        called_station_id=instance.called_station_id,
         calling_station_id=instance.calling_station_id,
         username=instance.username,
     )
+    logger.info(f'open_sessions: {open_sessions}')
     for session in open_sessions:
         session.stop_time = session.update_time
```

Comment:
@nemesifier in our testing we are not sending a RADIUS packet (`Interim-Update`) which sets the `update_time` field. Since, the `update_time` field is also set to `None`, the `stop_time` also stays `None`.

Reply:
ok so let's check if `update_time` is None and in that case let's set the current time on it, therefore `stop_time` should follow suit.